src/Razor/test/Microsoft.AspNetCore.Razor.Test.Common/Language/IntegrationTests/SourceMappingsSerializer.cs

```diff
@@ -34,7 +34,7 @@ private static void AppendMappingLocation(StringBuilder builder, SourceSpan loca
         {
             builder
                 .AppendLine(location.ToString())
-                .Append("|");
+                .Append('|');
```

Comment:
Is this to say the builder prefers appending a char over appending a string? Given a new string would be allocated in the end regardless, I found this surprising. Or perhaps this is just for inherent memory savings of a char over a string?

Reply:
I assume it's the inherent memory savings, and yeah there was just an analyzer complaining about this.